components/Text/Text.js

```diff
@@ -3,36 +3,34 @@ import * as styles from "./Text.module.scss";
 export default function Text({ mode, style, children }) {
   function styleClass() {
     switch (style) {
```

Comment:
```suggestion
  function styleClass() {
    const baseClass = `text`;
    const sizeClass = `text--${size}`;
```

Perhaps we can do this instead